{
  "uid" : "ebbfa13922b763a3",
  "name" : "新增用户",
  "fullName" : "test_userManage.TestUserManage#test_add_user",
  "historyId" : "61ff9853ff44167bffd24c6e86063367",
  "time" : {
    "start" : 1604654461283,
    "stop" : 1604654461296,
    "duration" : 13
  },
  "status" : "broken",
  "statusMessage" : "AttributeError: 'dict' object has no attribute 'json'",
  "statusTrace" : "self = <tc.test_userManage.TestUserManage object at 0x03FAC628>\ninData = '{\"password\":\"\",\"password_confirm\": \"88888888\",\"username\":\"001\"}'\nexp = '{\\'code\\':500,\\'msg\\':\"Key:\\'userAddRequest.password\\'Error:Field validation for \\'password\\' failed on the\\'required\\' tag\"}'\n\n    @allure.title(\"新增用户\")\n    @pytest.mark.parametrize(\"inData,exp\",get_excel_data(\"user\",2,6,7,8))\n    def test_add_user(self,inData,exp):\n        res = UserManage().add_user(self.token,inData)\n        print(inData)\n>       assert self.test.assert_code(res.json()[\"code\"],200)\nE       AttributeError: 'dict' object has no attribute 'json'\n\ntest_userManage.py:19: AttributeError",
  "flaky" : false,
  "newFailed" : false,
  "beforeStages" : [ {
    "name" : "_Class__pytest_setup_class",
    "time" : {
      "start" : 1604654461180,
      "stop" : 1604654461282,
      "duration" : 102
    },
    "status" : "passed",
    "steps" : [ ],
    "attachments" : [ ],
    "parameters" : [ ],
    "stepsCount" : 0,
    "hasContent" : false,
    "attachmentsCount" : 0,
    "shouldDisplayMessage" : false
  } ],
  "testStage" : {
    "status" : "broken",
    "statusMessage" : "AttributeError: 'dict' object has no attribute 'json'",
    "statusTrace" : "self = <tc.test_userManage.TestUserManage object at 0x03FAC628>\ninData = '{\"password\":\"\",\"password_confirm\": \"88888888\",\"username\":\"001\"}'\nexp = '{\\'code\\':500,\\'msg\\':\"Key:\\'userAddRequest.password\\'Error:Field validation for \\'password\\' failed on the\\'required\\' tag\"}'\n\n    @allure.title(\"新增用户\")\n    @pytest.mark.parametrize(\"inData,exp\",get_excel_data(\"user\",2,6,7,8))\n    def test_add_user(self,inData,exp):\n        res = UserManage().add_user(self.token,inData)\n        print(inData)\n>       assert self.test.assert_code(res.json()[\"code\"],200)\nE       AttributeError: 'dict' object has no attribute 'json'\n\ntest_userManage.py:19: AttributeError",
    "steps" : [ ],
    "attachments" : [ {
      "uid" : "62e5bc314e8ede52",
      "name" : "log",
      "source" : "62e5bc314e8ede52.txt",
      "type" : "text/plain",
      "size" : 490
    } ],
    "parameters" : [ ],
    "stepsCount" : 0,
    "hasContent" : true,
    "attachmentsCount" : 1,
    "shouldDisplayMessage" : true
  },
  "afterStages" : [ {
    "name" : "_Class__pytest_setup_class::0",
    "time" : {
      "start" : 1604654461466,
      "stop" : 1604654461466,
      "duration" : 0
    },
    "status" : "passed",
    "steps" : [ ],
    "attachments" : [ ],
    "parameters" : [ ],
    "stepsCount" : 0,
    "hasContent" : false,
    "attachmentsCount" : 0,
    "shouldDisplayMessage" : false
  } ],
  "labels" : [ {
    "name" : "suite",
    "value" : "test_userManage"
  }, {
    "name" : "subSuite",
    "value" : "TestUserManage"
  }, {
    "name" : "host",
    "value" : "dadaxPC008"
  }, {
    "name" : "thread",
    "value" : "15092-MainThread"
  }, {
    "name" : "framework",
    "value" : "pytest"
  }, {
    "name" : "language",
    "value" : "cpython3"
  }, {
    "name" : "package",
    "value" : "test_userManage"
  }, {
    "name" : "resultFormat",
    "value" : "allure2"
  } ],
  "parameters" : [ {
    "name" : "exp",
    "value" : "'{'code':500,'msg':\"Key:'userAddRequest.password'Error:Field validation for 'password' failed on the'required' tag\"}'"
  }, {
    "name" : "inData",
    "value" : "'{\"password\":\"\",\"password_confirm\": \"88888888\",\"username\":\"001\"}'"
  } ],
  "links" : [ ],
  "hidden" : true,
  "retry" : true,
  "extra" : {
    "categories" : [ ],
    "tags" : [ ]
  },
  "source" : "ebbfa13922b763a3.json",
  "parameterValues" : [ "'{'code':500,'msg':\"Key:'userAddRequest.password'Error:Field validation for 'password' failed on the'required' tag\"}'", "'{\"password\":\"\",\"password_confirm\": \"88888888\",\"username\":\"001\"}'" ]
}